install/swiftly-install.sh

```diff
@@ -127,29 +127,39 @@ install_system_deps () {
         return
     fi
 
-    dockerfile_url="https://raw.githubusercontent.com/apple/swift-docker/main/nightly-main/$docker_platform_name/$docker_platform_version/Dockerfile"
+    dockerfile_url="https://raw.githubusercontent.com/swiftlang/swift-docker/main/nightly-main/$docker_platform_name/$docker_platform_version/Dockerfile"
+    set +e
     dockerfile="$(curl --silent --retry 3 --location --fail $dockerfile_url)"
-    if [[ "$?" -ne 0 ]]; then
+    # if we couldn't find Dockerfile in nightly-main use swift-ci/master version
+    if [[ -z "$dockerfile" ]]; then
+        dockerfile_url="https://raw.githubusercontent.com/swiftlang/swift-docker/main/swift-ci/master/$docker_platform_name/$docker_platform_version/Dockerfile"
+        dockerfile="$(curl --silent --retry 3 --location --fail $dockerfile_url)"
```

Comment:
to avoid duplicating this incantation, can we factor it out to a function we can call here and above? we can also move the `set +e` and `set -e` in there too.

Reply:
Do you want the curl call and `set +e -e` in a separate function or the full test one location if it doesn't exist test the other location

Reply:
I was just thinking the curl and `set +e -e` so we could avoid duplicating the `curl` invocation. That way if we ever need to change the arguments or whatever we can do it in once place.